server/src/main/java/org/elasticsearch/ingest/IngestStats.java

```diff
@@ -38,8 +38,18 @@ public class IngestStats implements Writeable, ToXContentFragment {
      */
     public IngestStats(Stats totalStats, List<PipelineStat> pipelineStats, Map<String, List<ProcessorStat>> processorStats) {
         this.totalStats = totalStats;
-        this.pipelineStats = pipelineStats;
+        this.pipelineStats = pipelineStats.stream().sorted((p1, p2) -> {
+            final IngestStats.Stats p2Stats = p2.stats;
+            final IngestStats.Stats p1Stats = p1.stats;
+            final int ingestCountCompare = Long.compare(p2Stats.ingestCount, p1Stats.ingestCount);
+            if (ingestCountCompare == 0) {
+                return Long.compare(p2Stats.ingestTimeInMillis, p1Stats.ingestTimeInMillis);
```

Comment:
I wonder whether `ingestTimeInMillis` should be the primary sort property?
Because I think now pipelines that are used often but are relatively fast show up before pipelines that are used less often but are slower. I think the latter is a more important signal?

Reply:
Yeah I was on the fence about which one to be the primary and which one to be the fallback. I could go either way.

Reply:
Maybe @jakelandis would have an opinion here too, since he's looked more into ingest performance than I have

Reply:
I prefer `ingestTimeInMillis` to be the primary sort property in this case, because I think pipelines with higher time spent are more interesting to look at first irregardless of how often these pipelines were used.

Reply:
Okay, that works for me too, I'll swap the order here.